exoctk/exoctk_app/app_exoctk.py

```diff
@@ -510,11 +513,26 @@ def save_visib_result():
 
     visib_table = flask.request.form['data_file']
     targname = flask.request.form['targetname']
+    targname = targname.replace(' ', '_') # no spaces
     instname = flask.request.form['instrumentname']
 
     return flask.Response(visib_table, mimetype="text/dat",
                           headers={"Content-disposition": "attachment; filename={}_{}_visibility.csv".format(targname, instname)})
 
+@app_exoctk.route('/contam_verify', methods=['GET', 'POST'])
+def save_contam_pdf():
+    """Save the results of the Contamination Science FOV """
+
+    RA, DEC = '19:50:50.2400', '+48:04:51.00'
+    contam_pdf = contamVerify(RA, DEC, 'NIRISS', [1,2], binComp=[], PDF='', web=True)
+
+    filename = contam_pdf.split('/')[-1]
+    pdf_obj = PdfPages(contam_pdf)
+
+    return render_template(contam_pdf, filename, as_attachment=True)#, mimetype="application/pdf", as_attachment=True)
```

Comment:
Probably best to clean up commented out bits if they are not going to be used in production